deploy/014-riskfund-protocolshare.ts

```diff
@@ -11,6 +11,8 @@ import { Comptroller } from "../typechain";
 const MIN_AMOUNT_TO_CONVERT = convertToUnit(10, 18);
 const MIN_POOL_BAD_DEBT = convertToUnit(1000, 18);
 const maxLoopsLimit = 100;
+const waitForFirstBidder = 100; // for block based contracts deployment
+const nextBidderBlockOrTimestampLimit = 100; // for block based contracts deployment
```

Comment:
Related to SSV-01: we could move it to the `deploymentConfig.ts` file, and define it per network

Reply:
What would be the corresponding values for each network?.

Reply:
* Ethereum and Sepolia: BNB values divided by 4 (because on BNB there are 1 block every 3 seconds, on Ethereum there is a block every 12 seconds, and we could try to give to the bidders the same time (300 seconds))
* opBNB mainnet and testnet: BNB values multiplied by 3. On opBNB the block rate is 1 per second

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/324/commits/c749e0d6d301357876806274d244e06da0cb8108